src/resources/app/components/CourseCard.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+    <v-card
+      class="mx-auto rounded-3xl"
+      width="253"
+    >
+      <div
+        class="bg-primary h-24 font-semibold text-base text-slate-50"
+      >
+        <div class="flex flex-row">
+          <div 
+            class="pl-5 pt-5 text-base"
+            id="course-name"
+          >
+            Software Engineering
+          </div>
+          <button class="flex flex-col items-end space-x-0 space-y-0 pr-1 pt-5">
+            <div class="rotate-90 font-bold">
+              ...
+            </div>
+          </button>
+        </div>
+
+        <!--buletan syaiton-->
```

Comment:
wkwkwk should be omitted, or reworded to a more meaningful one

Reply:
oops, sorry, forgot about that one